tests/unit/auth/v4/headerAuthCheck.js

```diff
@@ -200,39 +187,19 @@ describe('v4 headerAuthCheck', () => {
                 '8e43e369f48ea6d3a5f1fe518e14',
         }, 'headers', request, headers);
         const clock = lolex.install(1454962445000);
-        headerAuthCheck(alteredRequest, log, err => {
-            clock.uninstall();
-            assert.ifError(err);
-            done();
-        });
+        const res = headerAuthCheck(alteredRequest, log);
+        clock.uninstall();
+        assert.ifError(res.err);
+        done();
     });
 
-    it('should successfully authenticate', done => {
+    it('should successfully return information', done => {
```

Comment:
Yes, in fact checking there is no error is enought, but we still check the good work of this function, not really relate to the version